src/main/java/net/helpscout/api/ApiClient.java

```diff
@@ -52,16 +55,44 @@
 
     private final JsonParser jsonParser = new JsonParser();
 
+    private LoggingInterceptor logInterceptor;
+    private OkHttpClient okHttpClient;
+    private Retrofit retrofit;
+    private ApiDeclaration apiClient;
+
+    private ApiClient() {
+        httpMethodWrapper.setBaseUrl(DEFAULT_BASE_URL);
+        logInterceptor = new LoggingInterceptor("some key");
```

Comment:
It is just some test of using Retrofit, so do not pay much attention to places like this.:)